src/m365/aad/commands/user/user-get.ts

```diff
@@ -131,7 +132,8 @@ class AadUserGetCommand extends GraphCommand {
       }
 
       if (res.value.length > 1) {
-        throw `Multiple users with ${identifier} found. Please disambiguate (user names): ${res.value.map(a => a.userPrincipalName).join(', ')} or (ids): ${res.value.map(a => a.id).join(', ')}`;
+        const resultAsKeyValuePair = formatting.convertArrayToHashTable('id', res.value);
+        res.value[0] = await Cli.handleMultipleResultsFound<User>(`Multiple users with ${identifier} found.`, resultAsKeyValuePair);
       }
 
       await logger.log(res.value[0]);
```

Comment:
How about this?
```suggestion
      if (res.value.length === 1) {
        await logger.log(res.value[0]);
      }

      const resultAsKeyValuePair = formatting.convertArrayToHashTable('id', res.value);
      const result = await Cli.handleMultipleResultsFound<User>(`Multiple users with ${identifier} found.`, resultAsKeyValuePair);
      await logger.log(result);
```

Reply:
What if it will be less than 1 😅. I would rather consider multiple results handling as exceptional and wrap this around a condition rather than the other way around

Reply:
The < 1 check is already done in the file on line 130... so this would seem perfectly fine. 
of course, what way you use the if-statement is not really important. Feel free to do it like this:

```typescript
if (res.value.length > 1) {
  const resultAsKeyValuePair = formatting.convertArrayToHashTable('id', res.value);
  const result = await Cli.handleMultipleResultsFound<User>(`Multiple users with ${identifier} found.`, resultAsKeyValuePair);
  await logger.log(result);
}

await logger.log(res.value[0]);
```

I find my suggestion slightly better to read but it's okay either way. The main point is that I think we should refrain from updating arrays by principle...